pkg/errs/errs.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 PingCAP, Inc.
```

Comment:
This file does not seem to include previous error types

Reply:
> This file does not seem to include previous error types

Yes. I think maybe define error by the package is a better choice?

Reply:
Maybe we should deal with this error file together at last.

Reply:
Of course